backend/internal/handlers/file_tree/service.go

```diff
@@ -0,0 +1,15 @@
+package file_tree
+
+import (
+	"github.com/CamPlume1/khoury-classroom/internal/github"
+	"github.com/CamPlume1/khoury-classroom/internal/storage"
+)
+
+type FileTreeService struct {
```

Comment:
Does this need to be a separate service? It seems like this is still a "github" related feature